xcodebuild/test_test.go

```diff
@@ -151,12 +143,29 @@ func TestTestCommandModel_cmdSlice(t *testing.T) {
 				"",
 			},
 		},
+		{
+			name:                      "SPM project",
+			projectPath:               "Package.swift",
+			scheme:                    "CoolLibrary",
+			destination:               "",
+			generateCodeCoverage:      false,
+			disableIndexWhileBuilding: false,
+			customBuildActions:        []string{""},
+			customOptions:             []string{""},
+			want: []string{
+				"xcodebuild",
+				"-scheme",
+				"CoolLibrary",
+				"",
```

Comment:
NOTE: I know that this is unrelated to the change, but don't we want to get rid of these empty command arguments?

Reply:
What caused is that we supplied an array of one item instead of an empty array in the test case. The logic than thought that the empty string is a valid extra parameter which needs to be included in the list of parameters. I have removed these.